src/YQUI.cc

```diff
@@ -608,6 +612,43 @@ bool YQUI::close()
 }
 
 
+void YQUI::askSendWidgetID()
+{
+    QWidget * parent = 0;
+    YDialog * dialog = YDialog::currentDialog( false );; // doThrow
+
+    if ( dialog )
+        parent = (QWidget *) dialog->widgetRep();
+
+    QString id = QInputDialog::getText( parent,
+                                        _( "Widget ID" ), // dialog title
+                                        _( "Enter Widget ID:" ) // label
+                                        );
+    if ( ! id.isEmpty() )
+    {
+        try
+        {
+            YWidget * widget = sendWidgetID( toUTF8( id ) );
+            YQGenericButton * yqButton = dynamic_cast<YQGenericButton *>( widget );
+
+            if ( yqButton )
+            {
+                yuiMilestone() << "Activating " << widget << endl;
+                yqButton->activate();
+            }
+        }
+        catch ( YUIWidgetNotFoundException & ex )
+        {
+            YUI_CAUGHT( ex );
+            QMessageBox::warning( parent,
+                                  "Error", // title
```

Comment:
The popup with widget ID uses translated strings so we should use translations in the error popup as well.